Makefile

```diff
@@ -5,23 +5,31 @@ UID = $(shell id -u)
 BUILD_FOLDER = $(PWD)/build/$(NAMESPACE)
 ROOT_DIR = $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
+BUILD_TYPE := $(or ${BUILD_TYPE},standalone)
 TARGET := $(or ${TARGET},minikube)
 PROFILE := $(or $(PROFILE),minikube)
 KUBECTL=kubectl -n $(NAMESPACE)
 
+ifeq ($(BUILD_TYPE), standalone)
```

Comment:
@YuviGold can you please change the BUILD_TYPE of the CI?
We can't pass the unit-tests for this PR until then since it doesn't create the coverage fi